src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DatabaseQueries/DatabaseQueriesInPXGraphInitializationTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Acuminator.Analyzers.StaticAnalysis;
+using Acuminator.Analyzers.StaticAnalysis.DatabaseQueries;
+using Acuminator.Analyzers.StaticAnalysis.PXGraph;
+using Acuminator.Tests.Helpers;
+using Acuminator.Tests.Verification;
+using Acuminator.Utilities;
+using Microsoft.CodeAnalysis.Diagnostics;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Acuminator.Tests.Tests.StaticAnalysis.DatabaseQueries
+{
+    public class DatabaseQueriesInPXGraphInitializationTests : DiagnosticVerifier
+    {
+        protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer() =>
+            new PXGraphAnalyzer(
+                CodeAnalysisSettings.Default
```

Comment:
Don't you need `WithRecursiveAnalysisEnabled()` here? Tests should not rely on the default settings because they can change in the future; all settings that are used in the tests must be set explicitly.

Reply:
Ok